pkg/middleware/stored_session.go

```diff
@@ -73,7 +73,10 @@ func (s *storedSessionLoader) loadSession(next http.Handler) http.Handler {
 			// In the case when there was an error loading the session,
 			// we should clear the session
 			logger.Printf("Error loading cookied session: %v, removing session", err)
-			s.store.Clear(rw, req)
+			err = s.store.Clear(rw, req)
+			if err != nil {
+				logger.Printf("Error removing session: %v", err)
```

Comment:
I was torn on this one: do we just log or fail the whole proxy if this aspect is breaking?

Reply:
I think log on this one

The alternative is that we could also return an internal server error, that would make it obvious that something broke, but might be bad for UX, since presently we would send the user to the login page if this log is returned right?